Cool_Retro_Term not installing on Twister OS using Raspberry Pi 400

### Confirmations

- [X] I searched for similar bug reports in https://github.com/Botspot/pi-apps/issues and found none were relevant.
- [X] I tried updating Pi Apps and the issue is still not fixed. [How to update Pi Apps?](https://github.com/Botspot/pi-apps#to-update-pi-apps)

### What happened?

App was installing and then stopped and siad there was a problem and a link directed me to GitHub

### Description

This is from the terminal where it installed. I'm no expert with this stuff, I tried to install it five different time before going to GitHub.

    ▅▅▅▅▅▅
 ▂▂▂█▂▂▂▂█▂▂▂
 ▌  ▗▖▗▖▗▖  ▐   █▀▀▆ ▄
 ▌  ▗▖▗▖▗▖  ▐   █▄▄▂ ▄ ▄▄ ▆▀▀█ █▀▀▆ █▀▀▆ ▆━━━
 ▌  ▗▖▗▖▗▖  ▐   █    █    ▂▄▄█ █▄▄▂ █▄▄▂ ▄▄▄▂
 ▂▃▃▃▃▃▃▃▃▃▃▂                  █    █

Installing Cool Retro Term...

Will install these packages: build-essential qmlscene qt5-qmake qtbase5-dev qtchooser qtdeclarative5-dev qml-module-qtquick-controls qml-module-qtgraphicaleffects qml-module-qtquick-dialogs qml-module-qtquick-localstorage qml-module-qtquick-window2 qml-module-qt-labs-settings qml-module-qt-labs-folderlistmodel 
Running sudo apt update...
Hit:1 http://archive.raspberrypi.org/debian buster InRelease
Hit:2 http://raspbian.raspberrypi.org/raspbian buster InRelease
Hit:3 http://pipplware.pplware.pt/pipplware/dists/buster/main/binary ./ InRelease
Ign:4 https://repo.vivaldi.com/stable/deb stable InRelease
Hit:5 https://repo.vivaldi.com/stable/deb stable Release
4 packages can be upgraded. Run 'apt list --upgradable' to see them.
apt update complete.
Some packages can be upgraded. Please consider running sudo apt full-upgrade.
Creating an empty apt-package to install the necessary apt packages...
It will be named: pi-apps-dca7a948
Depends: build-essential, qml-module-qtgraphicaleffects, qml-module-qt-labs-folderlistmodel, qml-module-qt-labs-settings, qml-module-qtquick-controls, qml-module-qtquick-dialogs, qml-module-qtquick-localstorage, qml-module-qtquick-window2, qmlscene, qt5-qmake, qtbase5-dev, qtchooser, qtdeclarative5-dev
Installing the pi-apps-dca7a948 package...
The following additional packages will be installed:
  libqt5quickparticles5 libqt5quicktest5 libqt5quickwidgets5
  qml-module-qt-labs-folderlistmodel qml-module-qt-labs-settings
  qml-module-qtquick-controls qml-module-qtquick-dialogs
  qml-module-qtquick-localstorage qml-module-qtquick-privatewidgets qmlscene
  qt5-qmake qt5-qmake-bin qtbase5-dev qtbase5-dev-tools qtdeclarative5-dev
  qtdeclarative5-dev-tools
Suggested packages:
  default-libmysqlclient-dev firebird-dev libpq-dev libsqlite3-dev unixodbc-dev
Recommended packages:
  libqt5opengl5-dev
The following NEW packages will be installed:
  libqt5quickparticles5 libqt5quicktest5 libqt5quickwidgets5 pi-apps-dca7a948
  qml-module-qt-labs-folderlistmodel qml-module-qt-labs-settings
  qml-module-qtquick-controls qml-module-qtquick-dialogs
  qml-module-qtquick-localstorage qml-module-qtquick-privatewidgets qmlscene
  qt5-qmake qt5-qmake-bin qtbase5-dev qtbase5-dev-tools qtdeclarative5-dev
  qtdeclarative5-dev-tools
0 upgraded, 17 newly installed, 0 to remove and 4 not upgraded.
Get:1 /home/pi/pi-apps-dca7a948.deb pi-apps-dca7a948 all 1.0 [880 B]
Get:2 http://mirror.pit.teraswitch.com/raspbian/raspbian buster/main armhf qml-module-qtquick-controls armhf 5.11.3-2 [627 kB]
Get:3 http://mirrors.syringanetworks.net/raspbian/raspbian buster/main armhf libqt5quickparticles5 armhf 5.11.3-4 [158 kB]
Get:4 http://mirrors.syringanetworks.net/raspbian/raspbian buster/main armhf libqt5quicktest5 armhf 5.11.3-4 [68.7 kB]
Get:5 http://mirrors.syringanetworks.net/raspbian/raspbian buster/main armhf libqt5quickwidgets5 armhf 5.11.3-4 [46.5 kB]
Get:6 http://mirror.pit.teraswitch.com/raspbian/raspbian buster/main armhf qml-module-qt-labs-folderlistmodel armhf 5.11.3-4 [43.9 kB]
Get:8 http://mirror.pit.teraswitch.com/raspbian/raspbian buster/main armhf qml-module-qtquick-privatewidgets armhf 5.11.3-2 [38.0 kB]
Get:9 http://mirror.pit.teraswitch.com/raspbian/raspbian buster/main armhf qml-module-qtquick-dialogs armhf 5.11.3-2 [109 kB]
Get:10 http://mirror.pit.teraswitch.com/raspbian/raspbian buster/main armhf qml-module-qtquick-localstorage armhf 5.11.3-4 [42.1 kB]
Get:11 http://mirror.pit.teraswitch.com/raspbian/raspbian buster/main armhf qmlscene armhf 5.11.3-4 [42.4 kB]
Get:13 http://mirror.pit.teraswitch.com/raspbian/raspbian buster/main armhf qt5-qmake armhf 5.11.3+dfsg1-1+rpi1+deb10u4 [213 kB]
Get:12 http://mirror.us.leaseweb.net/raspbian/raspbian buster/main armhf qt5-qmake-bin armhf 5.11.3+dfsg1-1+rpi1+deb10u4 [780 kB]
Get:14 http://mirror.pit.teraswitch.com/raspbian/raspbian buster/main armhf qtbase5-dev-tools armhf 5.11.3+dfsg1-1+rpi1+deb10u4 [630 kB]
Get:17 http://mirror.umd.edu/raspbian/raspbian buster/main armhf qtdeclarative5-dev armhf 5.11.3-4 [277 kB]
Get:15 http://mirror.pit.teraswitch.com/raspbian/raspbian buster/main armhf qtbase5-dev armhf 5.11.3+dfsg1-1+rpi1+deb10u4 [960 kB]
Get:16 http://mirror.pit.teraswitch.com/raspbian/raspbian buster/main armhf qtdeclarative5-dev-tools armhf 5.11.3-4 [300 kB]
Get:7 http://raspbian.mirror.constant.com/raspbian buster/main armhf qml-module-qt-labs-settings armhf 5.11.3-4 [37.0 kB]
Fetched 4,371 kB in 13s (329 kB/s)
(Reading database ... 317819 files and directories currently installed.)
Unpacking qml-module-qtquick-controls:armhf (5.11.3-2) ...
Unpacking libqt5quickparticles5:armhf (5.11.3-4) ...
Unpacking libqt5quicktest5:armhf (5.11.3-4) ...
Unpacking libqt5quickwidgets5:armhf (5.11.3-4) ...
Unpacking qml-module-qt-labs-folderlistmodel:armhf (5.11.3-4) ...
Unpacking qml-module-qt-labs-settings:armhf (5.11.3-4) ...
Unpacking qml-module-qtquick-privatewidgets:armhf (5.11.3-2) ...
Unpacking qml-module-qtquick-dialogs:armhf (5.11.3-2) ...
Unpacking qml-module-qtquick-localstorage:armhf (5.11.3-4) ...
Unpacking qmlscene (5.11.3-4) ...
Unpacking qt5-qmake-bin (5.11.3+dfsg1-1+rpi1+deb10u4) ...
Unpacking qt5-qmake:armhf (5.11.3+dfsg1-1+rpi1+deb10u4) ...
Unpacking qtbase5-dev-tools (5.11.3+dfsg1-1+rpi1+deb10u4) ...
Unpacking qtbase5-dev:armhf (5.11.3+dfsg1-1+rpi1+deb10u4) ...
Unpacking qtdeclarative5-dev-tools (5.11.3-4) ...
Unpacking qtdeclarative5-dev:armhf (5.11.3-4) ...
Unpacking pi-apps-dca7a948 (1.0) ...
Apt finished.
Package installation complete.
Will install these packages: qt5-default 
Running sudo apt update...
Hit:1 http://raspbian.raspberrypi.org/raspbian buster InRelease
Hit:2 http://archive.raspberrypi.org/debian buster InRelease
Hit:3 http://pipplware.pplware.pt/pipplware/dists/buster/main/binary ./ InRelease
Ign:4 https://repo.vivaldi.com/stable/deb stable InRelease
Hit:5 https://repo.vivaldi.com/stable/deb stable Release
4 packages can be upgraded. Run 'apt list --upgradable' to see them.
apt update complete.
Some packages can be upgraded. Please consider running sudo apt full-upgrade.
Creating an empty apt-package to install the necessary apt packages...
It will be named: pi-apps-dca7a948
The pi-apps-dca7a948 package is already installed. Inheriting its dependencies: build-essential, qml-module-qtgraphicaleffects, qml-module-qt-labs-folderlistmodel, qml-module-qt-labs-settings, qml-module-qtquick-controls, qml-module-qtquick-dialogs, qml-module-qtquick-localstorage, qml-module-qtquick-window2, qmlscene, qt5-qmake, qtbase5-dev, qtchooser, qtdeclarative5-dev
Depends: build-essential, qml-module-qtgraphicaleffects, qml-module-qt-labs-folderlistmodel, qml-module-qt-labs-settings, qml-module-qtquick-controls, qml-module-qtquick-dialogs, qml-module-qtquick-localstorage, qml-module-qtquick-window2, qmlscene, qt5-default, qt5-qmake, qtbase5-dev, qtchooser, qtdeclarative5-dev
Installing the pi-apps-dca7a948 package...
The following additional packages will be installed:
  qt5-default
The following NEW packages will be installed:
  qt5-default
The following packages will be upgraded:
  pi-apps-dca7a948
1 upgraded, 1 newly installed, 0 to remove and 4 not upgraded.
Get:1 /home/pi/pi-apps-dca7a948.deb pi-apps-dca7a948 all 1.0 [880 B]
Get:2 http://mirrors.syringanetworks.net/raspbian/raspbian buster/main armhf qt5-default armhf 5.11.3+dfsg1-1+rpi1+deb10u4 [48.5 kB]
apt-listchanges: Reading changelogs...
Fetched 48.5 kB in 2s (32.0 kB/s)
(Reading database ... 320941 files and directories currently installed.)
Unpacking qt5-default:armhf (5.11.3+dfsg1-1+rpi1+deb10u4) ...
Unpacking pi-apps-dca7a948 (1.0) over (1.0) ...
Apt finished.
Package installation complete.
Downloading cool-retro-term repository... Done
Info: creating stash file /home/pi/cool-retro-term/.qmake.stash
cd qmltermwidget/ && ( test -e Makefile || /usr/lib/qt5/bin/qmake -o Makefile /home/pi/cool-retro-term/qmltermwidget/qmltermwidget.pro ) && make -f Makefile 
make[1]: Entering directory '/home/pi/cool-retro-term/qmltermwidget'
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o BlockArray.o lib/BlockArray.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o ColorScheme.o lib/ColorScheme.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o Emulation.o lib/Emulation.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o Filter.o lib/Filter.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o History.o lib/History.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o HistorySearch.o lib/HistorySearch.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o KeyboardTranslator.o lib/KeyboardTranslator.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o konsole_wcwidth.o lib/konsole_wcwidth.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o kprocess.o lib/kprocess.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o kpty.o lib/kpty.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o kptydevice.o lib/kptydevice.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o kptyprocess.o lib/kptyprocess.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o ProcessInfo.o lib/ProcessInfo.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o Pty.o lib/Pty.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o Screen.o lib/Screen.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o ScreenWindow.o lib/ScreenWindow.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o Session.o lib/Session.cpp
In file included from lib/Session.cpp:43:
lib/TerminalDisplay.h:150:5: warning: "/*" within comment [-Wcomment]
     /** Sets the background image of the terminal display. */
      
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o ShellCommand.o lib/ShellCommand.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o TerminalCharacterDecoder.o lib/TerminalCharacterDecoder.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o TerminalDisplay.o lib/TerminalDisplay.cpp
In file included from lib/TerminalDisplay.cpp:24:
lib/TerminalDisplay.h:150:5: warning: "/*" within comment [-Wcomment]
     /** Sets the background image of the terminal display. */
      
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o tools.o lib/tools.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o Vt102Emulation.o lib/Vt102Emulation.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o qmltermwidget_plugin.o src/qmltermwidget_plugin.cpp
In file included from src/qmltermwidget_plugin.cpp:3:
lib/TerminalDisplay.h:150:5: warning: "/*" within comment [-Wcomment]
     /** Sets the background image of the terminal display. */
      
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o ksession.o src/ksession.cpp
g++ -pipe -O2 -dM -E -o moc_predefs.h /usr/lib/arm-linux-gnueabihf/qt5/mkspecs/features/data/dummy.cpp
/usr/lib/qt5/bin/moc -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB --include /home/pi/cool-retro-term/qmltermwidget/moc_predefs.h -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -I/home/pi/cool-retro-term/qmltermwidget -I/home/pi/cool-retro-term/qmltermwidget/lib -I/home/pi/cool-retro-term/qmltermwidget/src -I/usr/include/arm-linux-gnueabihf/qt5 -I/usr/include/arm-linux-gnueabihf/qt5/QtQuick -I/usr/include/arm-linux-gnueabihf/qt5/QtWidgets -I/usr/include/arm-linux-gnueabihf/qt5/QtGui -I/usr/include/arm-linux-gnueabihf/qt5/QtQml -I/usr/include/arm-linux-gnueabihf/qt5/QtNetwork -I/usr/include/arm-linux-gnueabihf/qt5/QtCore -I/usr/include/c++/8 -I/usr/include/arm-linux-gnueabihf/c++/8 -I/usr/include/c++/8/backward -I/usr/lib/gcc/arm-linux-gnueabihf/8/include -I/usr/local/include -I/usr/lib/gcc/arm-linux-gnueabihf/8/include-fixed -I/usr/include/arm-linux-gnueabihf -I/usr/include lib/Emulation.h -o moc_Emulation.cpp
/usr/lib/qt5/bin/moc -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB --include /home/pi/cool-retro-term/qmltermwidget/moc_predefs.h -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -I/home/pi/cool-retro-term/qmltermwidget -I/home/pi/cool-retro-term/qmltermwidget/lib -I/home/pi/cool-retro-term/qmltermwidget/src -I/usr/include/arm-linux-gnueabihf/qt5 -I/usr/include/arm-linux-gnueabihf/qt5/QtQuick -I/usr/include/arm-linux-gnueabihf/qt5/QtWidgets -I/usr/include/arm-linux-gnueabihf/qt5/QtGui -I/usr/include/arm-linux-gnueabihf/qt5/QtQml -I/usr/include/arm-linux-gnueabihf/qt5/QtNetwork -I/usr/include/arm-linux-gnueabihf/qt5/QtCore -I/usr/include/c++/8 -I/usr/include/arm-linux-gnueabihf/c++/8 -I/usr/include/c++/8/backward -I/usr/lib/gcc/arm-linux-gnueabihf/8/include -I/usr/local/include -I/usr/lib/gcc/arm-linux-gnueabihf/8/include-fixed -I/usr/include/arm-linux-gnueabihf -I/usr/include lib/Filter.h -o moc_Filter.cpp
/usr/lib/qt5/bin/moc -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB --include /home/pi/cool-retro-term/qmltermwidget/moc_predefs.h -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -I/home/pi/cool-retro-term/qmltermwidget -I/home/pi/cool-retro-term/qmltermwidget/lib -I/home/pi/cool-retro-term/qmltermwidget/src -I/usr/include/arm-linux-gnueabihf/qt5 -I/usr/include/arm-linux-gnueabihf/qt5/QtQuick -I/usr/include/arm-linux-gnueabihf/qt5/QtWidgets -I/usr/include/arm-linux-gnueabihf/qt5/QtGui -I/usr/include/arm-linux-gnueabihf/qt5/QtQml -I/usr/include/arm-linux-gnueabihf/qt5/QtNetwork -I/usr/include/arm-linux-gnueabihf/qt5/QtCore -I/usr/include/c++/8 -I/usr/include/arm-linux-gnueabihf/c++/8 -I/usr/include/c++/8/backward -I/usr/lib/gcc/arm-linux-gnueabihf/8/include -I/usr/local/include -I/usr/lib/gcc/arm-linux-gnueabihf/8/include-fixed -I/usr/include/arm-linux-gnueabihf -I/usr/include lib/HistorySearch.h -o moc_HistorySearch.cpp
/usr/lib/qt5/bin/moc -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB --include /home/pi/cool-retro-term/qmltermwidget/moc_predefs.h -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -I/home/pi/cool-retro-term/qmltermwidget -I/home/pi/cool-retro-term/qmltermwidget/lib -I/home/pi/cool-retro-term/qmltermwidget/src -I/usr/include/arm-linux-gnueabihf/qt5 -I/usr/include/arm-linux-gnueabihf/qt5/QtQuick -I/usr/include/arm-linux-gnueabihf/qt5/QtWidgets -I/usr/include/arm-linux-gnueabihf/qt5/QtGui -I/usr/include/arm-linux-gnueabihf/qt5/QtQml -I/usr/include/arm-linux-gnueabihf/qt5/QtNetwork -I/usr/include/arm-linux-gnueabihf/qt5/QtCore -I/usr/include/c++/8 -I/usr/include/arm-linux-gnueabihf/c++/8 -I/usr/include/c++/8/backward -I/usr/lib/gcc/arm-linux-gnueabihf/8/include -I/usr/local/include -I/usr/lib/gcc/arm-linux-gnueabihf/8/include-fixed -I/usr/include/arm-linux-gnueabihf -I/usr/include lib/kprocess.h -o moc_kprocess.cpp
/usr/lib/qt5/bin/moc -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB --include /home/pi/cool-retro-term/qmltermwidget/moc_predefs.h -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -I/home/pi/cool-retro-term/qmltermwidget -I/home/pi/cool-retro-term/qmltermwidget/lib -I/home/pi/cool-retro-term/qmltermwidget/src -I/usr/include/arm-linux-gnueabihf/qt5 -I/usr/include/arm-linux-gnueabihf/qt5/QtQuick -I/usr/include/arm-linux-gnueabihf/qt5/QtWidgets -I/usr/include/arm-linux-gnueabihf/qt5/QtGui -I/usr/include/arm-linux-gnueabihf/qt5/QtQml -I/usr/include/arm-linux-gnueabihf/qt5/QtNetwork -I/usr/include/arm-linux-gnueabihf/qt5/QtCore -I/usr/include/c++/8 -I/usr/include/arm-linux-gnueabihf/c++/8 -I/usr/include/c++/8/backward -I/usr/lib/gcc/arm-linux-gnueabihf/8/include -I/usr/local/include -I/usr/lib/gcc/arm-linux-gnueabihf/8/include-fixed -I/usr/include/arm-linux-gnueabihf -I/usr/include lib/kptydevice.h -o moc_kptydevice.cpp
lib/TerminalDisplay.h: In constructor ‘Konsole::TerminalDisplay::TerminalDisplay(QQuickItem*)’:
lib/TerminalDisplay.h:928:10: warning: ‘Konsole::TerminalDisplay::m_full_cursor_height’ will be initialized after [-Wreorder]
     bool m_full_cursor_height;
          ^~~~~~~~~~~~~~~~~~~~
lib/TerminalDisplay.h:920:9: warning:   ‘int Konsole::TerminalDisplay::_leftBaseMargin’ [-Wreorder]
     int _leftBaseMargin;
         ^~~~~~~~~~~~~~~
lib/TerminalDisplay.cpp:320:1: warning:   when initialized here [-Wreorder]
 TerminalDisplay::TerminalDisplay(QQuickItem *parent)
 ^~~~~~~~~~~~~~~
/usr/lib/qt5/bin/moc -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB --include /home/pi/cool-retro-term/qmltermwidget/moc_predefs.h -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -I/home/pi/cool-retro-term/qmltermwidget -I/home/pi/cool-retro-term/qmltermwidget/lib -I/home/pi/cool-retro-term/qmltermwidget/src -I/usr/include/arm-linux-gnueabihf/qt5 -I/usr/include/arm-linux-gnueabihf/qt5/QtQuick -I/usr/include/arm-linux-gnueabihf/qt5/QtWidgets -I/usr/include/arm-linux-gnueabihf/qt5/QtGui -I/usr/include/arm-linux-gnueabihf/qt5/QtQml -I/usr/include/arm-linux-gnueabihf/qt5/QtNetwork -I/usr/include/arm-linux-gnueabihf/qt5/QtCore -I/usr/include/c++/8 -I/usr/include/arm-linux-gnueabihf/c++/8 -I/usr/include/c++/8/backward -I/usr/lib/gcc/arm-linux-gnueabihf/8/include -I/usr/local/include -I/usr/lib/gcc/arm-linux-gnueabihf/8/include-fixed -I/usr/include/arm-linux-gnueabihf -I/usr/include lib/kptyprocess.h -o moc_kptyprocess.cpp
/usr/lib/qt5/bin/moc -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB --include /home/pi/cool-retro-term/qmltermwidget/moc_predefs.h -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -I/home/pi/cool-retro-term/qmltermwidget -I/home/pi/cool-retro-term/qmltermwidget/lib -I/home/pi/cool-retro-term/qmltermwidget/src -I/usr/include/arm-linux-gnueabihf/qt5 -I/usr/include/arm-linux-gnueabihf/qt5/QtQuick -I/usr/include/arm-linux-gnueabihf/qt5/QtWidgets -I/usr/include/arm-linux-gnueabihf/qt5/QtGui -I/usr/include/arm-linux-gnueabihf/qt5/QtQml -I/usr/include/arm-linux-gnueabihf/qt5/QtNetwork -I/usr/include/arm-linux-gnueabihf/qt5/QtCore -I/usr/include/c++/8 -I/usr/include/arm-linux-gnueabihf/c++/8 -I/usr/include/c++/8/backward -I/usr/lib/gcc/arm-linux-gnueabihf/8/include -I/usr/local/include -I/usr/lib/gcc/arm-linux-gnueabihf/8/include-fixed -I/usr/include/arm-linux-gnueabihf -I/usr/include lib/Pty.h -o moc_Pty.cpp
/usr/lib/qt5/bin/moc -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB --include /home/pi/cool-retro-term/qmltermwidget/moc_predefs.h -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -I/home/pi/cool-retro-term/qmltermwidget -I/home/pi/cool-retro-term/qmltermwidget/lib -I/home/pi/cool-retro-term/qmltermwidget/src -I/usr/include/arm-linux-gnueabihf/qt5 -I/usr/include/arm-linux-gnueabihf/qt5/QtQuick -I/usr/include/arm-linux-gnueabihf/qt5/QtWidgets -I/usr/include/arm-linux-gnueabihf/qt5/QtGui -I/usr/include/arm-linux-gnueabihf/qt5/QtQml -I/usr/include/arm-linux-gnueabihf/qt5/QtNetwork -I/usr/include/arm-linux-gnueabihf/qt5/QtCore -I/usr/include/c++/8 -I/usr/include/arm-linux-gnueabihf/c++/8 -I/usr/include/c++/8/backward -I/usr/lib/gcc/arm-linux-gnueabihf/8/include -I/usr/local/include -I/usr/lib/gcc/arm-linux-gnueabihf/8/include-fixed -I/usr/include/arm-linux-gnueabihf -I/usr/include lib/ScreenWindow.h -o moc_ScreenWindow.cpp
/usr/lib/qt5/bin/moc -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB --include /home/pi/cool-retro-term/qmltermwidget/moc_predefs.h -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -I/home/pi/cool-retro-term/qmltermwidget -I/home/pi/cool-retro-term/qmltermwidget/lib -I/home/pi/cool-retro-term/qmltermwidget/src -I/usr/include/arm-linux-gnueabihf/qt5 -I/usr/include/arm-linux-gnueabihf/qt5/QtQuick -I/usr/include/arm-linux-gnueabihf/qt5/QtWidgets -I/usr/include/arm-linux-gnueabihf/qt5/QtGui -I/usr/include/arm-linux-gnueabihf/qt5/QtQml -I/usr/include/arm-linux-gnueabihf/qt5/QtNetwork -I/usr/include/arm-linux-gnueabihf/qt5/QtCore -I/usr/include/c++/8 -I/usr/include/arm-linux-gnueabihf/c++/8 -I/usr/include/c++/8/backward -I/usr/lib/gcc/arm-linux-gnueabihf/8/include -I/usr/local/include -I/usr/lib/gcc/arm-linux-gnueabihf/8/include-fixed -I/usr/include/arm-linux-gnueabihf -I/usr/include lib/Session.h -o moc_Session.cpp
/usr/lib/qt5/bin/moc -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB --include /home/pi/cool-retro-term/qmltermwidget/moc_predefs.h -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -I/home/pi/cool-retro-term/qmltermwidget -I/home/pi/cool-retro-term/qmltermwidget/lib -I/home/pi/cool-retro-term/qmltermwidget/src -I/usr/include/arm-linux-gnueabihf/qt5 -I/usr/include/arm-linux-gnueabihf/qt5/QtQuick -I/usr/include/arm-linux-gnueabihf/qt5/QtWidgets -I/usr/include/arm-linux-gnueabihf/qt5/QtGui -I/usr/include/arm-linux-gnueabihf/qt5/QtQml -I/usr/include/arm-linux-gnueabihf/qt5/QtNetwork -I/usr/include/arm-linux-gnueabihf/qt5/QtCore -I/usr/include/c++/8 -I/usr/include/arm-linux-gnueabihf/c++/8 -I/usr/include/c++/8/backward -I/usr/lib/gcc/arm-linux-gnueabihf/8/include -I/usr/local/include -I/usr/lib/gcc/arm-linux-gnueabihf/8/include-fixed -I/usr/include/arm-linux-gnueabihf -I/usr/include lib/TerminalDisplay.h -o moc_TerminalDisplay.cpp
/usr/lib/qt5/bin/moc -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB --include /home/pi/cool-retro-term/qmltermwidget/moc_predefs.h -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -I/home/pi/cool-retro-term/qmltermwidget -I/home/pi/cool-retro-term/qmltermwidget/lib -I/home/pi/cool-retro-term/qmltermwidget/src -I/usr/include/arm-linux-gnueabihf/qt5 -I/usr/include/arm-linux-gnueabihf/qt5/QtQuick -I/usr/include/arm-linux-gnueabihf/qt5/QtWidgets -I/usr/include/arm-linux-gnueabihf/qt5/QtGui -I/usr/include/arm-linux-gnueabihf/qt5/QtQml -I/usr/include/arm-linux-gnueabihf/qt5/QtNetwork -I/usr/include/arm-linux-gnueabihf/qt5/QtCore -I/usr/include/c++/8 -I/usr/include/arm-linux-gnueabihf/c++/8 -I/usr/include/c++/8/backward -I/usr/lib/gcc/arm-linux-gnueabihf/8/include -I/usr/local/include -I/usr/lib/gcc/arm-linux-gnueabihf/8/include-fixed -I/usr/include/arm-linux-gnueabihf -I/usr/include lib/Vt102Emulation.h -o moc_Vt102Emulation.cpp
/usr/lib/qt5/bin/moc -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB --include /home/pi/cool-retro-term/qmltermwidget/moc_predefs.h -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -I/home/pi/cool-retro-term/qmltermwidget -I/home/pi/cool-retro-term/qmltermwidget/lib -I/home/pi/cool-retro-term/qmltermwidget/src -I/usr/include/arm-linux-gnueabihf/qt5 -I/usr/include/arm-linux-gnueabihf/qt5/QtQuick -I/usr/include/arm-linux-gnueabihf/qt5/QtWidgets -I/usr/include/arm-linux-gnueabihf/qt5/QtGui -I/usr/include/arm-linux-gnueabihf/qt5/QtQml -I/usr/include/arm-linux-gnueabihf/qt5/QtNetwork -I/usr/include/arm-linux-gnueabihf/qt5/QtCore -I/usr/include/c++/8 -I/usr/include/arm-linux-gnueabihf/c++/8 -I/usr/include/c++/8/backward -I/usr/lib/gcc/arm-linux-gnueabihf/8/include -I/usr/local/include -I/usr/lib/gcc/arm-linux-gnueabihf/8/include-fixed -I/usr/include/arm-linux-gnueabihf -I/usr/include src/qmltermwidget_plugin.h -o moc_qmltermwidget_plugin.cpp
lib/TerminalDisplay.cpp: In member function ‘virtual bool Konsole::TerminalDisplay::event(QEvent*)’:
lib/TerminalDisplay.cpp:3039:54: warning: this statement may fall through [-Wimplicit-fallthrough=]
         _scrollBar->setPalette( QApplication::palette() );
                                 ~~~~~~~~~~~~~~~~~~~~~^~
lib/TerminalDisplay.cpp:3040:5: note: here
     case QEvent::InputMethodQuery:
     ^~~~
/usr/lib/qt5/bin/moc -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB --include /home/pi/cool-retro-term/qmltermwidget/moc_predefs.h -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -I/home/pi/cool-retro-term/qmltermwidget -I/home/pi/cool-retro-term/qmltermwidget/lib -I/home/pi/cool-retro-term/qmltermwidget/src -I/usr/include/arm-linux-gnueabihf/qt5 -I/usr/include/arm-linux-gnueabihf/qt5/QtQuick -I/usr/include/arm-linux-gnueabihf/qt5/QtWidgets -I/usr/include/arm-linux-gnueabihf/qt5/QtGui -I/usr/include/arm-linux-gnueabihf/qt5/QtQml -I/usr/include/arm-linux-gnueabihf/qt5/QtNetwork -I/usr/include/arm-linux-gnueabihf/qt5/QtCore -I/usr/include/c++/8 -I/usr/include/arm-linux-gnueabihf/c++/8 -I/usr/include/c++/8/backward -I/usr/lib/gcc/arm-linux-gnueabihf/8/include -I/usr/local/include -I/usr/lib/gcc/arm-linux-gnueabihf/8/include-fixed -I/usr/include/arm-linux-gnueabihf -I/usr/include src/ksession.h -o moc_ksession.cpp
lib/Vt102Emulation.cpp: In member function ‘virtual void Konsole::Vt102Emulation::sendMouseEvent(int, int, int, int)’:
lib/Vt102Emulation.cpp:971:44: warning: ‘%d’ directive output may be truncated writing between 1 and 10 bytes into a region of size between 7 and 26 [-Wformat-truncation=]
         snprintf(command, sizeof(command), "\033[%d;%d;%dM", cb + 0x20, cx, cy);
                                            ^~~~~~~~~~~~~~~~
lib/Vt102Emulation.cpp:971:44: note: directive argument in the range [1, 2147483647]
lib/Vt102Emulation.cpp:971:17: note: ‘snprintf’ output between 9 and 37 bytes into a destination of size 32
         snprintf(command, sizeof(command), "\033[%d;%d;%dM", cb + 0x20, cx, cy);
         ~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o moc_Emulation.o moc_Emulation.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o moc_Filter.o moc_Filter.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o moc_HistorySearch.o moc_HistorySearch.cpp
src/ksession.cpp: In member function ‘void KSession::sendKey(int, int, int) const’:
src/ksession.cpp:239:28: warning: unused parameter ‘rep’ [-Wunused-parameter]
 void KSession::sendKey(int rep, int key, int mod) const
                        ~~~~^~~
src/ksession.cpp:239:37: warning: unused parameter ‘key’ [-Wunused-parameter]
 void KSession::sendKey(int rep, int key, int mod) const
                                 ~~~~^~~
src/ksession.cpp:239:46: warning: unused parameter ‘mod’ [-Wunused-parameter]
 void KSession::sendKey(int rep, int key, int mod) const
                                          ~~~~^~~
src/ksession.h:40: Warning: Property declaration shellProgram has no READ accessor function or associated MEMBER variable. The property will be invalid.
src/ksession.h:41: Warning: Property declaration shellProgramArgs has no READ accessor function or associated MEMBER variable. The property will be invalid.
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o moc_kprocess.o moc_kprocess.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o moc_kptydevice.o moc_kptydevice.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o moc_kptyprocess.o moc_kptyprocess.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o moc_Pty.o moc_Pty.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o moc_ScreenWindow.o moc_ScreenWindow.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o moc_Session.o moc_Session.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o moc_TerminalDisplay.o moc_TerminalDisplay.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o moc_Vt102Emulation.o moc_Vt102Emulation.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o moc_qmltermwidget_plugin.o moc_qmltermwidget_plugin.cpp
g++ -c -pipe -O2 -D_REENTRANT -Wall -W -fPIC -DHAVE_POSIX_OPENPT -DHAVE_SYS_TIME_H -DQT_NO_DEBUG -DQT_PLUGIN -DQT_QUICK_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -Ilib -Isrc -isystem /usr/include/arm-linux-gnueabihf/qt5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQuick -isystem /usr/include/arm-linux-gnueabihf/qt5/QtWidgets -isystem /usr/include/arm-linux-gnueabihf/qt5/QtGui -isystem /usr/include/arm-linux-gnueabihf/qt5/QtQml -isystem /usr/include/arm-linux-gnueabihf/qt5/QtNetwork -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore -I. -I/usr/lib/arm-linux-gnueabihf/qt5/mkspecs/linux-g++ -o moc_ksession.o moc_ksession.cpp
In file included from moc_TerminalDisplay.cpp:9:
lib/TerminalDisplay.h:150:5: warning: "/*" within comment [-Wcomment]
     /** Sets the background image of the terminal display. */
      
rm -f libqmltermwidget.so
g++ -Wl,-O1 -shared -o libqmltermwidget.so BlockArray.o ColorScheme.o Emulation.o Filter.o History.o HistorySearch.o KeyboardTranslator.o konsole_wcwidth.o kprocess.o kpty.o kptydevice.o kptyprocess.o ProcessInfo.o Pty.o Screen.o ScreenWindow.o Session.o ShellCommand.o TerminalCharacterDecoder.o TerminalDisplay.o tools.o Vt102Emulation.o qmltermwidget_plugin.o ksession.o moc_Emulation.o moc_Filter.o moc_HistorySearch.o moc_kprocess.o moc_kptydevice.o moc_kptyprocess.o moc_Pty.o moc_ScreenWindow.o moc_Session.o moc_TerminalDisplay.o moc_Vt102Emulation.o moc_qmltermwidget_plugin.o moc_ksession.o  -lQt5Quick -lQt5Widgets -lQt5Gui -lQt5Qml -lQt5Network -lQt5Core -lGLESv2 -lpthread -latomic  
mv -f libqmltermwidget.so QMLTermWidget/libqmltermwidget.so
cp -f -R /home/pi/cool-retro-term/qmltermwidget/lib/color-schemes /home/pi/cool-retro-term/qmltermwidget/QMLTermWidget && cp -f -R /home/pi/cool-retro-term/qmltermwidget/lib/kb-layouts /home/pi/cool-retro-term/qmltermwidget/QMLTermWidget && cp -f /home/pi/cool-retro-term/qmltermwidget/src/qmldir /home/pi/cool-retro-term/qmltermwidget/QMLTermWidget && cp -f /home/pi/cool-retro-term/qmltermwidget/src/QMLTermScrollbar.qml /home/pi/cool-retro-term/qmltermwidget/QMLTermWidget
make[1]: Leaving directory '/home/pi/cool-retro-term/qmltermwidget'
cd app/ && ( test -e Makefile || /usr/lib/qt5/bin/qmake -o Makefile /home/pi/cool-retro-term/app/app.pro ) && make -f Makefile 
Project ERROR: Unknown module(s) in QT: quickcontrols2
make: *** [Makefile:89: sub-app-make_first-ordered] Error 3
failed on make step!
Allowing packages required by the Cool Retro Term app to be uninstalled
These packages were: build-essential, qml-module-qtgraphicaleffects, qml-module-qt-labs-folderlistmodel, qml-module-qt-labs-settings, qml-module-qtquick-controls, qml-module-qtquick-dialogs, qml-module-qtquick-localstorage, qml-module-qtquick-window2, qmlscene, qt5-default, qt5-qmake, qtbase5-dev, qtchooser, qtdeclarative5-dev
Purging the pi-apps-dca7a948 package...
The following packages will be REMOVED:
  libqt5quickparticles5* libqt5quicktest5* libqt5quickwidgets5*
  pi-apps-dca7a948* qml-module-qt-labs-folderlistmodel*
  qml-module-qt-labs-settings* qml-module-qtquick-controls*
  qml-module-qtquick-dialogs* qml-module-qtquick-localstorage*
  qml-module-qtquick-privatewidgets* qmlscene* qt5-default* qt5-qmake*
  qt5-qmake-bin* qtbase5-dev* qtbase5-dev-tools* qtdeclarative5-dev*
  qtdeclarative5-dev-tools*
0 upgraded, 0 newly installed, 18 to remove and 4 not upgraded.
                                                               (Reading database(Reading database ... 320945 files and directories currently installed.)
Removing pi-apps-dca7a948 (1.0) ...
Removing qtdeclarative5-dev:armhf (5.11.3-4) ...
Removing libqt5quickparticles5:armhf (5.11.3-4) ...
Removing qtdeclarative5-dev-tools (5.11.3-4) ...
Removing libqt5quicktest5:armhf (5.11.3-4) ...
Removing libqt5quickwidgets5:armhf (5.11.3-4) ...
Removing qml-module-qt-labs-folderlistmodel:armhf (5.11.3-4) ...
Removing qml-module-qt-labs-settings:armhf (5.11.3-4) ...
Removing qml-module-qtquick-controls:armhf (5.11.3-2) ...
Removing qml-module-qtquick-dialogs:armhf (5.11.3-2) ...
Removing qml-module-qtquick-localstorage:armhf (5.11.3-4) ...
Removing qml-module-qtquick-privatewidgets:armhf (5.11.3-2) ...
Removing qmlscene (5.11.3-4) ...
Removing qt5-default:armhf (5.11.3+dfsg1-1+rpi1+deb10u4) ...
Removing qtbase5-dev:armhf (5.11.3+dfsg1-1+rpi1+deb10u4) ...
Removing qt5-qmake:armhf (5.11.3+dfsg1-1+rpi1+deb10u4) ...
Removing qt5-qmake-bin (5.11.3+dfsg1-1+rpi1+deb10u4) ...
Removing qtbase5-dev-tools (5.11.3+dfsg1-1+rpi1+deb10u4) ...
Apt finished.
All packages have been purged succesfully.

Failed to install Cool Retro Term!
◢◣Need help? Copy the ENTIRE terminal output or take a screenshot.
Please ask on Github: https://github.com/Botspot/pi-apps/issues/new/choose
Or on Discord: https://discord.gg/RXSTvaUvuu

Close this window to exit.



### What is your operating system?

I am using Twister OS with the Raspberry Pi 400

### (Optional) Error log? Terminal output? Debug messages?

```bash
*Failed to install Cool Retro Term!

*This is the only error message I had found
```

here is the error:
```
Project ERROR: Unknown module(s) in QT: quickcontrols2
make: *** [Makefile:89: sub-app-make_first-ordered] Error 3
failed on make step!
```

--- a/apps/Cool Retro Term/install
+++ b/apps/Cool Retro Term/install
@@ -1,6 +1,6 @@
 #!/bin/bash
 # Get dependencies
-install_packages build-essential qmlscene qt5-qmake qtbase5-dev qtchooser qtdeclarative5-dev qml-module-qtquick-controls qml-module-qtgraphicaleffects qml-module-qtquick-dialogs qml-module-qtquick-localstorage qml-module-qtquick-window2 qml-module-qt-labs-settings qml-module-qt-labs-folderlistmodel || error "Failed to install dependencies"
+install_packages build-essential qmlscene qt5-qmake qtbase5-dev qtchooser qtdeclarative5-dev qtquickcontrols2-5-dev qml-module-qtquick-controls qml-module-qtgraphicaleffects qml-module-qtquick-dialogs qml-module-qtquick-localstorage qml-module-qtquick-window2 qml-module-qt-labs-settings qml-module-qt-labs-folderlistmodel || error "Failed to install dependencies"
 # install qt5-default if its availabe in apt (necessary on debian/ubuntu versions prior to bullseye/hirsute)
 package_available qt5-default
 if [[ $? == "0" ]]; then

